woocommerce/woocommerce-ios
# [Mobile Payments] Add card images to Refund Confirmation screen

Closes: #6179 


Will not merge until after 8.6 freezes – there's too much to tweak with the designs still.

### Description

Adds card brand images to the Refund Via row in the Refund Confirmation screen. This is to fulfil brand guidelines from card networks, and to make it easier to identify the card in question when performing a refund.

Card images were taken from the `Hardware` target and are not final – e.g. we do not have an `Interac` image, and the `Visa` image is outdated (it should not have a gold flash any more.)

Apologies for the length - it's mostly SVGs

### Testing instructions


1. In the orders tab, open an order which was paid for with a Card Present payment
2. Tap `Issue Refund`
3. Select some of the order to refund, and tap `Next`
4. Observe that the card image is shown alongside the card details
5. With VoiceOver on, hear that the payment method cell is read out as `Visa card ending four two four two`

### Screenshots

![Card image on refund confirmation](https://user-images.githubusercontent.com/2472348/154724225-85db247e-2493-4c77-bc3d-0f9a856ae518.png)

![card-brand-screenshot-interac](https://user-images.githubusercontent.com/2472348/154941561-0a969f1a-1b39-4584-89a6-42500a54b119.png) | ![card-brand-screenshot-visa](https://user-images.githubusercontent.com/2472348/154941569-d9929d9a-f22e-4823-a195-405b292e157f.png) | ![card-brand-screenshot-amex](https://user-images.githubusercontent.com/2472348/154941570-525a469c-5a64-43e1-9899-a3750b4de54f.png)
![card-brand-screenshot-diners](https://user-images.githubusercontent.com/2472348/154941571-8bcd9c44-4e3b-4168-abc4-aa60cd3ee9ba.png) | ![card-brand-screenshot-discover](https://user-images.githubusercontent.com/2472348/154941573-6bd19473-cb8f-4857-8e21-7fce7efa4f9e.png) | ![card-brand-screenshot-jcb](https://user-images.githubusercontent.com/2472348/154941576-938a8a49-ac14-4e5c-93db-85ae1c39c721.png)
![card-brand-screenshot-mastercard](https://user-images.githubusercontent.com/2472348/154984673-f81545d6-b2cf-4c4a-af67-2a16a250f79f.png) | ![card-brand-screenshot-unionpay](https://user-images.githubusercontent.com/2472348/154941580-5c59d8b5-f08b-40de-bf38-bcdd18538840.png) | ![card-brand-screenshot-unknown](https://user-images.githubusercontent.com/2472348/154941582-071d70f5-69df-447c-b452-68d02f14f387.png)


### Design review questions
#### Visa 
This should not have the gold flash seen in the screenshot above any more. [The Visa brand guidelines page](https://merchantsignageap.visa.com/apac_brand_guidelines) provides SVGZ files, but I can't find a free way to turn those in to SVGs for use in the app – Adobe CC will do it I think. Please could you get the up to date logo as an SVG for me?

#### Interac
I've used the [Interac logo SVG from WikiMedia](https://commons.wikimedia.org/wiki/File:InteracLogo.svg) – I'm not sure whether we actually want to do that, but it was provided by Interac and it's in line with their [brand guidelines](https://interacposcentre.ca/images/INTERAC_Brand_Guidelines_Vendors_Apr2021_FINAL.pdf), except that it includes a ® stamp.

#### Discover and Diners club
The brand guidelines suggest that the images we have (taken from WCPay) are outdated. There are EPS files for both [available to download on discoverglobalsinage.com](https://discoverglobalsignage.com/free-signage-logos). I think the correct ones are the Acceptance Marks using RGB colour. Similar to Visa, I don't have any tools suitable for doing the conversion to SVG myself: would you be able to convert them please?

#### Amex & UnionPay
I wasn't able to find useful branding guidelines for these two card networks, but I think what we have here is appropriate. An internal Amex document suggested that the `AMEX` square should only be used at very small sizes, and I think that the size we have here is appropriate.

#### Brand guidelines used
Where I could find them, I checked our use of these marks against the brand guidelines that are available.

- [Discover and Diners Club](https://discoverglobalsignage.com/uploads/09-12-21_03:59_DGN_BG_10052021_AccMk.pdf)
- [Mastercard](https://brand.mastercard.com/brandcenter/branding-requirements/mastercard.html)
- [Visa](https://spponeimages.azureedge.net/prod/a8c3df05-c491-404b-b41b-e6bfe0d77e3b2a_VDB-experience-guidelines-2018.pdf)
- [JCB](https://www.global.jcb/en/about-us/brand-concept/agreement/index.html)


---
